setup.cfg

```
@@ -9,6 +9,24 @@ project_urls =
 description_file = README.md
 description_content_type = text/markdown
 license_file = LICENSE.md
+license = Apache
```

There is no standard here, and the way the pypi UI shows this is basically `license (classifier-license)`.

But after looking through the list of packages with the apache classifier, I've come to believe `Apache Software License, Version 2.0` covers the most.

```suggestion
license = Apache Software License, Version 2.0
```